test/unit/api-client/api-client.submit.spec.ts

```diff
@@ -203,19 +243,7 @@ describe('api-client job submission', () => {
 
         it('submit job with all options null', async () => {
```

Comment:
More accurate test name
```suggestion
        it('submit job with all options undefined', async () => {
```